docs/cli/edgedb.rst

```diff
@@ -23,17 +23,17 @@ queries and seeing results interactively.
 Options
 =======
 
-:cli:synopsis:`-?, --help`
+:cli:synopsis:`-h, --help`
```

Comment:
I would keep `-?` as well, its a common convention on Windows.

Reply:
On Unixes it must be quoted because it's a shell pattern. Isn't it on new windows terminal is the same too? (It looks' like `?` is a [globbing pattern on powershell](https://www.leeholmes.com/blog/2015/01/23/the-wonderful-world-of-filtering-and-globbing/)). And rust implementation will need more work for having another alias for help command (I know that's unfortunate). So I had no motivation to do that. Also on windows it's likely `/?`, no?

Do you still think it worth it?

Reply:
OK, let's drop it.